libs/sql-schema-describer/src/postgres.rs

```diff
@@ -442,7 +442,9 @@ impl<'a> super::SqlSchemaDescriberBackend for SqlSchemaDescriber<'a> {
 
     async fn get_metadata(&self, schema: &str) -> DescriberResult<SqlMetadata> {
         let mut sql_schema = SqlSchema::default();
-        sql_schema.push_namespace((*schema).into());
+
+        // TODO(MultiSchema): is this correct?
+        self.get_namespaces(&mut sql_schema, &[schema]).await?;
```

Comment:
get_metadata is barely used if at all in the CLI (I think one command we don't document), so we needn't worry too much